com.unity.render-pipelines.universal/Editor/ShaderPreprocessor.cs

```diff
@@ -40,10 +50,26 @@ internal class ShaderPreprocessor : IPreprocessShaders
         ShaderKeyword m_AlphaTestOn = new ShaderKeyword("_ALPHATEST_ON");
         ShaderKeyword m_GbufferNormalsOct = new ShaderKeyword("_GBUFFER_NORMALS_OCT");
 
-        ShaderKeyword m_DeprecatedVertexLights = new ShaderKeyword("_VERTEX_LIGHTS");
-        ShaderKeyword m_DeprecatedShadowsEnabled = new ShaderKeyword("_SHADOWS_ENABLED");
-        ShaderKeyword m_DeprecatedShadowsCascade = new ShaderKeyword("_SHADOWS_CASCADE");
-        ShaderKeyword m_DeprecatedLocalShadowsEnabled = new ShaderKeyword("_LOCAL_SHADOWS_ENABLED");
+        ShaderKeyword Feature00 = new ShaderKeyword("_FEATURE00");
```

Comment:
Do you mean to have these features added to this?

Reply:
No, that's Draft PR, it's not ready for review it. I just opened it so I can diff easier code to make tests ;)